.vsts-ci.yml

```diff
@@ -348,11 +348,10 @@ stages:
 
   - template: /src/SourceBuild/Arcade/eng/common/templates/job/source-build-create-tarball.yml
 
-  # disable tarball build ci until 7.0 stabilizes
-  # - template: /src/SourceBuild/Arcade/eng/common/templates/job/source-build-run-tarball-build.yml
-  #   parameters:
-  #     dependsOn: Source_Build_Create_Tarball
-  #     condition: eq(dependencies.Source_Build_Create_Tarball.outputs['Tarball_Build_Check._includeTarballBuild'], 'true')
+  - template: /src/SourceBuild/Arcade/eng/common/templates/job/source-build-run-tarball-build.yml
```

Comment:
We are going to want to disable offline in PR validation.  To do that, move [this line](https://github.com/dotnet/installer/blob/main/src/SourceBuild/Arcade/eng/common/templates/job/source-build-run-tarball-build.yml#L51) up so that CentOS7-Offline is within the condition.